packages/go/cypher/frontend/query.go

```diff
@@ -363,43 +362,34 @@ func NewMultiPartQueryVisitor() *MultiPartQueryVisitor {
 }
 
 func (s *MultiPartQueryVisitor) EnterOC_ReadingClause(ctx *parser.OC_ReadingClauseContext) {
-	s.currentPart = model.NewMultiPartQueryPart()
-	s.Query.Parts = append(s.Query.Parts, s.currentPart)
-
 	s.ctx.Enter(NewReadingClauseVisitor())
 }
 
 func (s *MultiPartQueryVisitor) ExitOC_ReadingClause(ctx *parser.OC_ReadingClauseContext) {
-	if s.currentPart != nil {
-		s.currentPart.AddReadingClause(s.ctx.Exit().(*ReadingClauseVisitor).ReadingClause)
-	} else {
-		s.ctx.AddErrors(ErrInvalidInput)
-	}
+	part := model.NewMultiPartQueryPart()
+	part.AddReadingClause(s.ctx.Exit().(*ReadingClauseVisitor).ReadingClause)
+	s.Query.Parts = append(s.Query.Parts, part)
 }
 
 func (s *MultiPartQueryVisitor) EnterOC_UpdatingClause(ctx *parser.OC_UpdatingClauseContext) {
 	s.ctx.Enter(NewUpdatingClauseVisitor())
 }
 
 func (s *MultiPartQueryVisitor) ExitOC_UpdatingClause(ctx *parser.OC_UpdatingClauseContext) {
```

Comment:
What does "OC" stand for?

Reply:
I'm not entirely sure. These are all overrides for functions that are defined by the antlr generated code

Reply:
Cool was just wondering since I didn't see it spelled out anywhere at a glance.